tutorials/rust/baked/build.rs

```diff
@@ -12,12 +12,15 @@ fn main() {
     let mod_directory = PathBuf::from(std::env::var_os("OUT_DIR").unwrap()).join("baked_data");
 
     DatagenDriver::new()
-        .with_locales([langid!("ru")])
+        .with_locales_and_fallback([LocaleFamily::single(langid!("ru"))], {
+            let mut options = FallbackOptions::default();
+            options.runtime_fallback_location = Some(RuntimeFallbackLocation::External);
+            options
+        })
```

Comment:
Issue: `RuntimeManual` maps to `LocaleFamily::with_descendants`, `DeduplicationStrategy::Maximal`, and `RuntimeFallbackLocation::External`

Reply:
doesn't matter, it's an example

Reply:
ok... except if you use `LocaleFamily::single`, that's really a power user feature, so it probably shouldn't be highlighted in the tutorial

Reply:
Can we resolve tutorial nits later and unblock the release?

Reply:
Ok